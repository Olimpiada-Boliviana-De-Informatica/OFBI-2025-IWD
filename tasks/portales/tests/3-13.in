75
722179894 712881005 855302835 857236183 965404596 255501497 733146893 963985179 907869901 984490438 40262487 322278917 480402642 117075631 106922690 996758116 128986615 807372821 646966949 458082762 120917716 216270751 926566138 83335951 783615717 737035892 313009886 987068110 623819019 898860994 56122133 189576678 42583050 36699963 598570627 536577018 749951753 487201171 773625012 439708658 715669012 238990083 755373141 38197368 781396637 349116361 439251384 300683900 451426295 765035686 60071764 684864433 882553818 22582622 660729411 83801357 6635746 135644988 878163651 740703565 388789520 91638155 735655094 39466862 160611399 911557150 398231585 100212021 665155280 973256649 913608112 595021224 19427287 561801957 782626609