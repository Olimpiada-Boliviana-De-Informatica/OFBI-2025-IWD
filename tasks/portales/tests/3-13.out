-1
-1
722179894
855302835
857236183
-1
722179894
857236183
857236183
965404596
-1
255501497
322278917
40262487
40262487
984490438
117075631
733146893
480402642
322278917
117075631
128986615
907869901
40262487
733146893
733146893
255501497
984490438
480402642
857236183
40262487
128986615
40262487
-1
480402642
480402642
737035892
480402642
749951753
322278917
712881005
216270751
749951753
36699963
773625012
322278917
349116361
255501497
439708658
755373141
56122133
646966949
857236183
-1
646966949
83335951
-1
128986615
857236183
737035892
349116361
83801357
733146893
38197368
135644988
907869901
388789520
91638155
660729411
965404596
911557150
536577018
6635746
536577018
781396637
